src/main/scala/com/gd/twitteranalytics/TwitterReportingJob.scala

```diff
@@ -0,0 +1,55 @@
+package com.gd.twitteranalytics
+
+import com.gd.twitteranalytics.util.AppConfigReader
+import org.apache.hadoop.fs.{FileSystem, Path}
+import org.apache.log4j.Logger
+import org.apache.spark.sql.SparkSession
+
+object TwitterReportingJob {
```

Comment:
there are going to be many different reports.
please apply appropriate naming facilitating extension.